apps/Launcher/mainwindow.cpp

```diff
@@ -263,3 +329,20 @@ void MainWindow::newsReadyRead() {
     _informationWidget->setText(news);
     _newsReply->deleteLater();
 }
+
+void MainWindow::generateOptions() {
+
+    _optionParser = new QCommandLineParser;
+    _optionParser->setApplicationDescription("OpenSpace Launcher");
+    _optionParser->addHelpOption();
+    _optionParser->addVersionOption();
+
+    _optionParser->addOptions({
+        { { "d", "debug" },
+        QCoreApplication::translate("main", "Debug output level"),
+        QCoreApplication::translate("main", "0, 1, 2, 3, 4, 5, 6"),
+        QCoreApplication::translate("main", "1"),
+        }
```

Comment:
Indeed it should. I think I pulled it out while I was testing things out and never put it back in. Thanks for catching it!